From 723d4bf71957586591a2adf15f701b8f052f1d34 Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Thu, 18 Jan 2018 22:05:58 +0000
Subject: [PATCH] MANTA-3563 Muppet fails to build when cloned using public
 GitHub URL Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
 Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

---
 .gitmodules | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 7b4be7d..dff00c3 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,15 +1,15 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = git://github.com/trentm/restdown.git
+	url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/manta-scripts"]
 	path = deps/manta-scripts
-	url = git@github.com:joyent/manta-scripts.git
+	url = https://github.com/joyent/manta-scripts.git
 [submodule "deps/haproxy-1.5"]
 	path = deps/haproxy-1.5
 	url = https://github.com/joyent/haproxy-1.5.git
-- 
2.21.0

